### ISSUE TITLE
$ref to adjacent_file object in #/components/parameters block not resolving

### ISSUE THREAD

**Issue by seahorseing-around** [Mermade/openapi-lint-vscode#23]
Loving the plugin! making lightweight of a new complex spec - but run into something you might not have caught yet.

I have [references ](https://swagger.io/docs/specification/using-ref/) to both schema and parameter objects in an adjacent file.

If the reference is to `adjacent_file.yaml#/components/schemas/schemaObject` the link **does** resolve 

If the reference is to `adjacent_file.yaml#/components/parameters/schemaObject` the link **does not** resolve 

I wonder if the plugin is only built to support the schemas block? I haven't tried the other available components sections: https://swagger.io/docs/specification/components/.

Thanks,
Craig

**Comment by MikeRalphson** [comment]
No, references should work wherever allowed by the spec. Can you share your files in a gist so I can debug?

**Comment by seahorseing-around** [comment]
Afraid i can't share them - but i can try to recreate on an example - i'll see what i can do.

**Comment by MikeRalphson** [comment]
Thanks. The extension uses the same libraries as `boast` in the `swagger2openapi` npm package. You could use that in verbose mode to see if there are any problems resolving your refs.

**Comment by seahorseing-around** [comment]
[SWAGGER Lint.zip](https://github.com/Mermade/openapi-lint-vscode/files/5062444/SWAGGER.Lint.zip)

Sorry - not used Gist before and saw an easy attach here option - hope this is ok. 

petstore_no_components.yaml presents an error: 
![image](https://user-images.githubusercontent.com/46077013/90011926-5bb6c380-dc9a-11ea-9034-0b872ed6119f.png)

If you uncomment the parameters block in petstore_no_components.yaml and switch the commented lines in the refs to use the local parameters all errors go, indicating the schemas references in the external file are resolving fine.

**Comment by MikeRalphson** [comment]
I'll admit this is a little confusing in the current version. The message means the reference cannot be resolved as an internal reference. The command you need to use is "OpenAPI Resolve and Lint" if your API definition is made of multiple parts. I'll probably soon release a new version where this is the default behaviour anyway.
